src/components/Settings/AllSettingsFields.tsx

```diff
@@ -264,6 +264,23 @@ export const AllSettingsFields = forwardRef(
               , and start a discussion if you don't see it! Your feedback will
               help us prioritize what to build next.
             </p>
+            {APP_NAME.indexOf('-nightly') === -1 && (
```

Comment:
I'm guess you'll probably want to see this at some global state level @franknoirot if we're going to enable in-development commands based on that

Reply:
Yeah but this is a solid pattern I can hoist up in a follow-up PR, appreciate it!